cfme/configure/configuration/__init__.py

```diff
@@ -294,230 +301,134 @@ def step(self):
 
-    @classmethod
-    def get_last_message(cls):
-        """ Returns the Last Message that is displayed in the InfoBlock.
+    @property
+    def last_message(self):
+        view = navigate_to(self.appliance.server, 'DiagnosticsCollectLogs')
+        return view.last_log_message.text
 
-        """
-        return cls.elements.last_message.text
+    @property
+    def is_cleared(self):
+        view = navigate_to(self.appliance.server, 'DiagnosticsCollectLogs')
+        return view.log_depot_uri.text == "N/A"
 
-    @classmethod
-    def get_last_collection(cls):
-        """ Returns the Last Log Collection that is displayed in the InfoBlock.
+    def clear(self):
+        """ Set depot type to "No Depot"
 
-        Returns: If it is Never, returns `None`, otherwise :py:class:`utils.timeutil.parsetime`.
         """
-        d = cls.elements.last_log_collection.text
-        if d.strip().lower() == "never":
-            return None
-        else:
-            try:
-                return parsetime.from_american_with_utc(d.strip())
-            except ValueError:
-                return parsetime.from_iso_with_utc(d.strip())
+        if not self.is_cleared:
+            view = navigate_to(self.appliance.server, 'DiagnosticsCollectLogsEdit')
+            if BZ.bugzilla.get_bug(1436326).is_opened:
+                wait_for(lambda: view.type.selected_option != '<No Depot>', num_sec=5)
+            view.type.fill('<No Depot>')
+            view.save.click()
+            view.flash.assert_success_message("Log Depot Settings were saved")
 
-    @classmethod
-    def _collect(cls, selection, wait_minutes=4):
-        """ Initiate and wait for collection to finish. DRY method.
+    def _collect(self, selection):
+        """ Initiate and wait for collection to finish.
 
         Args:
             selection: The item in Collect menu ('Collect all logs' or 'Collect current logs')
-            wait_minutes: How many minutes should we wait for the collection process to finish?
         """
-        navigate_to(current_appliance.server, 'DiagnosticsCollectLogs')
-        last_collection = cls.get_last_collection()
+
+        view = navigate_to(self.appliance.server, 'DiagnosticsCollectLogs')
+        last_collection = self.last_collection
         # Initiate the collection
         tb.select("Collect", selection)
-        flash.assert_no_errors()
+        view.flash.assert_success_message(
+            "Log collection for CFME MiqServer {} [{}] has been initiated".
+            format(self.appliance.server_name(), self.appliance.server_zone_id()))
 
         def _refresh():
             """ The page has no refresh button, so we'll switch between tabs.
 
-            Why this? Selenium's refresh() is way too slow. This is much faster.
-
             """
-            tabs.select_tab("Workers")
-            tabs.select_tab("Collect Logs")  # Serve as the refresh
+            navigate_to(self.appliance.server, 'Workers')
```

Comment:
please restore the comment explaining this behavior, we may later on want to implement it better